openapi/nytimes.mostpopular/types.bal

```diff
@@ -15,78 +15,78 @@
 // under the License.
 
 public type ArticlewithcounttypeMedia record {
-    string caption?;
-    string copyright?;
-    ArticlewithcounttypeMediametadata[] mediaMetadata?;
-    string subtype?;
-    string 'type?;
+    string? caption?;
```

Comment:
Fields are not originally nillable. Please regenerate without giving `--nullable` in the command.